packages/components/collapse/stories/Collapse.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import type { Story } from '@storybook/react/types-6-0';
+import { Stack } from '@contentful/f36-core';
+import { SectionHeading, Text } from '@contentful/f36-typography';
+import { Button } from '@contentful/f36-button';
+import { Collapse, CollapseProps } from '../src';
+
+export default {
+  title: 'Components/Collapse',
```

Comment:
```suggestion
  title: 'Animation/Collapse',
```

What do you think about creating a new section?

Reply:
ohhh that might be good. we could then also add like the turning of the chevron icon, the "drawer opening" from the sidebar etc.